tests/integration/resource_options/program/resource.ts

```diff
@@ -18,5 +18,8 @@ export class Provider implements pulumi.dynamic.ResourceProvider {
 export class Resource extends pulumi.dynamic.Resource {
     constructor(name: string, opts?: pulumi.CustomResourceOptions) {
         super(Provider.instance, name, {}, opts);
+        if (name === "aliased") {
+            console.log("OPTS = " + JSON.stringify(opts));
+        }
```

Comment:
Do we need to keep this? I assume it was to debug the alias thing per comments in the policy packs for this test.